app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalSearchController.php

```diff
@@ -63,31 +62,30 @@ public function get(Request $request): KnowledgeManagementSearchData
                 fn (Stringable $string) => $string->explode(',')
             );
 
-        $itemData = KnowledgeBaseArticleData::collection(
+        $itemData =
```

Comment:
I don't believe you need to return the DTO here, I think you can pass a paginator into a collection for a DTO: https://spatie.be/docs/laravel-data/v4/as-a-resource/from-data-to-resource#content-paginators